src/bundle/Resources/public/scss/_buttons.scss

```diff
@@ -69,3 +69,45 @@
     align-items: center;
     justify-content: center;
 }
+
+.ez-modal {
+    &--send-to-trash {
+        .modal-header {
+            .close {
+                position: absolute;
+                right: 1rem;
+            }
+        }
+
+        .modal-footer {
+            .btn {
+                line-height: 1.25;
+                padding: .5rem .75rem;
+            }
+
+            .btn-secondary {
+                background-color: $ez-color-base-medium;
+                border-color: $ez-color-base-medium;
+
+                &:hover {
+                    background-color: darken($ez-color-base-medium, 15%);
```

Comment:
Shouldn't we use colors defined in `_custom.scss`? Maybe we should new ones there?